<!DOCTYPE html>
<html>
<head>
<title>resume.md</title>
<meta http-equiv="Content-type" content="text/html;charset=UTF-8">

<style>
/* https://github.com/microsoft/vscode/blob/master/extensions/markdown-language-features/media/markdown.css */
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/

body {
	font-family: var(--vscode-markdown-font-family, -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif);
	font-size: var(--vscode-markdown-font-size, 14px);
	padding: 0 26px;
	line-height: var(--vscode-markdown-line-height, 22px);
	word-wrap: break-word;
}

#code-csp-warning {
	position: fixed;
	top: 0;
	right: 0;
	color: white;
	margin: 16px;
	text-align: center;
	font-size: 12px;
	font-family: sans-serif;
	background-color:#444444;
	cursor: pointer;
	padding: 6px;
	box-shadow: 1px 1px 1px rgba(0,0,0,.25);
}

#code-csp-warning:hover {
	text-decoration: none;
	background-color:#007acc;
	box-shadow: 2px 2px 2px rgba(0,0,0,.25);
}

body.scrollBeyondLastLine {
	margin-bottom: calc(100vh - 22px);
}

body.showEditorSelection .code-line {
	position: relative;
}

body.showEditorSelection .code-active-line:before,
body.showEditorSelection .code-line:hover:before {
	content: "";
	display: block;
	position: absolute;
	top: 0;
	left: -12px;
	height: 100%;
}

body.showEditorSelection li.code-active-line:before,
body.showEditorSelection li.code-line:hover:before {
	left: -30px;
}

.vscode-light.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(0, 0, 0, 0.15);
}

.vscode-light.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(0, 0, 0, 0.40);
}

.vscode-light.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-dark.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 255, 255, 0.4);
}

.vscode-dark.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 255, 255, 0.60);
}

.vscode-dark.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

.vscode-high-contrast.showEditorSelection .code-active-line:before {
	border-left: 3px solid rgba(255, 160, 0, 0.7);
}

.vscode-high-contrast.showEditorSelection .code-line:hover:before {
	border-left: 3px solid rgba(255, 160, 0, 1);
}

.vscode-high-contrast.showEditorSelection .code-line .code-line:hover:before {
	border-left: none;
}

img {
	max-width: 100%;
	max-height: 100%;
}

a {
	text-decoration: none;
}

a:hover {
	text-decoration: underline;
}

a:focus,
input:focus,
select:focus,
textarea:focus {
	outline: 1px solid -webkit-focus-ring-color;
	outline-offset: -1px;
}

hr {
	border: 0;
	height: 2px;
	border-bottom: 2px solid;
}

h1 {
	padding-bottom: 0.3em;
	line-height: 1.2;
	border-bottom-width: 1px;
	border-bottom-style: solid;
}

h1, h2, h3 {
	font-weight: normal;
}

table {
	border-collapse: collapse;
}

table > thead > tr > th {
	text-align: left;
	border-bottom: 1px solid;
}

table > thead > tr > th,
table > thead > tr > td,
table > tbody > tr > th,
table > tbody > tr > td {
	padding: 5px 10px;
}

table > tbody > tr + tr > td {
	border-top: 1px solid;
}

blockquote {
	margin: 0 7px 0 5px;
	padding: 0 16px 0 10px;
	border-left-width: 5px;
	border-left-style: solid;
}

code {
	font-family: Menlo, Monaco, Consolas, "Droid Sans Mono", "Courier New", monospace, "Droid Sans Fallback";
	font-size: 1em;
	line-height: 1.357em;
}

body.wordWrap pre {
	white-space: pre-wrap;
}

pre:not(.hljs),
pre.hljs code > div {
	padding: 16px;
	border-radius: 3px;
	overflow: auto;
}

pre code {
	color: var(--vscode-editor-foreground);
	tab-size: 4;
}

/** Theming */

.vscode-light pre {
	background-color: rgba(220, 220, 220, 0.4);
}

.vscode-dark pre {
	background-color: rgba(10, 10, 10, 0.4);
}

.vscode-high-contrast pre {
	background-color: rgb(0, 0, 0);
}

.vscode-high-contrast h1 {
	border-color: rgb(0, 0, 0);
}

.vscode-light table > thead > tr > th {
	border-color: rgba(0, 0, 0, 0.69);
}

.vscode-dark table > thead > tr > th {
	border-color: rgba(255, 255, 255, 0.69);
}

.vscode-light h1,
.vscode-light hr,
.vscode-light table > tbody > tr + tr > td {
	border-color: rgba(0, 0, 0, 0.18);
}

.vscode-dark h1,
.vscode-dark hr,
.vscode-dark table > tbody > tr + tr > td {
	border-color: rgba(255, 255, 255, 0.18);
}

</style>

<style>
/* Tomorrow Theme */
/* http://jmblog.github.com/color-themes-for-google-code-highlightjs */
/* Original theme - https://github.com/chriskempson/tomorrow-theme */

/* Tomorrow Comment */
.hljs-comment,
.hljs-quote {
	color: #8e908c;
}

/* Tomorrow Red */
.hljs-variable,
.hljs-template-variable,
.hljs-tag,
.hljs-name,
.hljs-selector-id,
.hljs-selector-class,
.hljs-regexp,
.hljs-deletion {
	color: #c82829;
}

/* Tomorrow Orange */
.hljs-number,
.hljs-built_in,
.hljs-builtin-name,
.hljs-literal,
.hljs-type,
.hljs-params,
.hljs-meta,
.hljs-link {
	color: #f5871f;
}

/* Tomorrow Yellow */
.hljs-attribute {
	color: #eab700;
}

/* Tomorrow Green */
.hljs-string,
.hljs-symbol,
.hljs-bullet,
.hljs-addition {
	color: #718c00;
}

/* Tomorrow Blue */
.hljs-title,
.hljs-section {
	color: #4271ae;
}

/* Tomorrow Purple */
.hljs-keyword,
.hljs-selector-tag {
	color: #8959a8;
}

.hljs {
	display: block;
	overflow-x: auto;
	color: #4d4d4c;
	padding: 0.5em;
}

.hljs-emphasis {
	font-style: italic;
}

.hljs-strong {
	font-weight: bold;
}
</style>

<style>
/*
 * Markdown PDF CSS
 */

 body {
	font-family: -apple-system, BlinkMacSystemFont, "Segoe WPC", "Segoe UI", "Ubuntu", "Droid Sans", sans-serif, "Meiryo";
	padding: 0 12px;
}

pre {
	background-color: #f8f8f8;
	border: 1px solid #cccccc;
	border-radius: 3px;
	overflow-x: auto;
	white-space: pre-wrap;
	overflow-wrap: break-word;
}

pre:not(.hljs) {
	padding: 23px;
	line-height: 19px;
}

blockquote {
	background: rgba(127, 127, 127, 0.1);
	border-color: rgba(0, 122, 204, 0.5);
}

.emoji {
	height: 1.4em;
}

code {
	font-size: 14px;
	line-height: 19px;
}

/* for inline code */
:not(pre):not(.hljs) > code {
	color: #C9AE75; /* Change the old color so it seems less like an error */
	font-size: inherit;
}

/* Page Break : use <div class="page"/> to insert page break
-------------------------------------------------------- */
.page {
	page-break-after: always;
}

</style>

<script src="https://unpkg.com/mermaid/dist/mermaid.min.js"></script>
</head>
<body>
  <script>
    mermaid.initialize({
      startOnLoad: true,
      theme: document.body.classList.contains('vscode-dark') || document.body.classList.contains('vscode-high-contrast')
          ? 'dark'
          : 'default'
    });
  </script>
<h1 id="niu-du-phd">Niu Du, PhD</h1>
<p><strong>Phone:</strong> +1 (858) 603-5663  | <strong>Email:</strong> ndu0328@gmail.com  | <a href="https://github.com/ndu-bioinfo">GitHub: https://github.com/ndu-bioinfo</a></p>
<h2 id="areas-of-expertise">Areas of Expertise</h2>
<p><strong>Bioinformatics &amp; Computational Biology</strong><br>
Expertise in multiomics data analysis and the integration of computational methods with biological insights.</p>
<p><strong>Data Science &amp; Statistics</strong><br>
Development and application of machine learning, AI models, and statistical methods for drug discovery.</p>
<p><strong>Pipeline Engineering</strong><br>
Design and implementation of robust, scalable pipelines for high-throughput data processing and quality control.</p>
<p><strong>Data Management &amp; Integration</strong>
Expertise in organizing, integrating, and maintaining large-scale datasets, including database management, API development, and efficient data retrieval to ensure seamless data access and use.</p>
<h2 id="technical-skills">Technical skills</h2>
<p><strong>Programming Languages</strong>
Python, R, SQL, Bash<br>
<strong>Cloud/Grid Computing</strong>
Experience with cloud platforms and grid HPC computing environments</p>
<h2 id="professional-experience">Professional Experience</h2>
<h3 id="visiting-bioinformatics-scientist">Visiting Bioinformatics Scientist</h3>
<p><strong>J. Craig Venter Institute</strong><br>
May 2024 – Present</p>
<ul>
<li>Developing a Bayesian statistics-based toolkit for drug discovery and clinical data analysis, specifically designed for undersampled datasets with outliers.</li>
</ul>
<h3 id="senior-bioinformatics-scientist">Senior Bioinformatics Scientist</h3>
<p><strong>Creyon Bio</strong><br>
September 2022 – May 2024</p>
<ul>
<li>Developed and implemented next-generation sequencing (NGS) pipelines on the Google Cloud Platform (GCP) for quality control, data management, and execution monitoring, tailored for drug discovery research.</li>
<li>Integrated pipelines with cloud platforms and developed APIs to ensure seamless data flow from storage to databases and dashboard visualizations, facilitating effective data science workflows.</li>
<li>Conducted data analyses to optimize pipeline performance and refine quality control outputs, enhancing accuracy and efficiency in identifying potential drug targets.</li>
<li>Developed machine learning algorithms for weight-based Negative Binomial likelihood estimation, improving the detection of active pharmaceutical ingredients in antisense oligonucleotide (ASO) drug development.</li>
<li>Managed Docker containers for pipeline modules and implemented CI/CD pipelines for package repositories, significantly improving deployment efficiency and reliability.</li>
<li>Evaluated and optimized sequencing platforms, leading troubleshooting efforts to improve sequencing reliability and operational efficiency, essential for high-throughput drug screening.</li>
<li>Collaborated with the sequencing team to troubleshoot experimental issues and develop custom Python tools, supporting both production and research and development (R&amp;D) operations in drug discovery.</li>
<li>Established design and technical guidelines for pipeline frameworks, standardizing implementation and operational best practices in bioinformatics and drug discovery projects.</li>
<li>Provided mentorship to junior bioinformatics scientists and created comprehensive onboarding documentation, streamlining team integration and enhancing productivity in data-driven drug discovery projects.</li>
</ul>
<h3 id="bioinformatics-engineer-algorithm">Bioinformatics Engineer, Algorithm</h3>
<p><strong>Invitae Corporation</strong><br>
June 2020 – September 2022</p>
<ul>
<li>Led the development of in vitro diagnostic (IVD) targeted DNA diagnostic assays for clinical use, contributing to successful product launches.</li>
<li>Developed an automated oligo design workflow, reducing sequencing costs by 45% and increasing operational efficiency.</li>
<li>Developed machine-learning-based target classification algorithms to improve assay specificity and sensitivity.</li>
<li>Directed the development of bioinformatics tools from concept to library integration, ensuring robustness and reliability through comprehensive testing.</li>
<li>Troubleshot pipeline operational issues and provided emergency response to address critical issues for customers, ensuring system stability and reliability.</li>
<li>Actively participated in code reviews, planning sessions, and best practices discussions, fostering a collaborative and high-quality development environment.</li>
</ul>
<h3 id="postdoctoral-fellow">Postdoctoral Fellow</h3>
<p><strong>La Jolla Institute for Immunology</strong><br>
April 2019 – June 2020</p>
<ul>
<li>Developed RNA-Seq mapping and QC pipelines for bulk and single-cell sequencing data.</li>
<li>Created CITE-seq pipelines for single-cell data preprocessing and clustering, improving data analysis workflows.</li>
<li>Collaborated with experimentalists on bulk and single-cell data analysis related to lung diseases.</li>
</ul>
<h3 id="postdoctoral-fellow">Postdoctoral Fellow</h3>
<p><strong>J. Craig Venter Institute</strong><br>
April 2018 – April 2019</p>
<ul>
<li>Developed analytical pipelines for 16S, 18S, and metatranscriptome data, as well as transcription factor-DNA-binding assays.</li>
<li>Conducted statistical analyses on human oral microbiome data for caries detection using mixed-effect models.</li>
</ul>
<h3 id="research-associate">Research Associate</h3>
<p><strong>J. Craig Venter Institute</strong><br>
2015 – 2017</p>
<ul>
<li>Investigated metabolic pathways in photosynthesis using genome-scale models, contributing to a deeper understanding of energy utilization.</li>
<li>Developed a novel method for rapid carbon fixation rate measurement, including system integration, protocol development, and software creation.</li>
<li>Created a deep learning model for two-factor RNA-seq data analysis, enhancing data interpretation capabilities.</li>
<li>Performed various wet lab experiments and routine bioinformatics tasks, including pipeline execution and server maintenance.</li>
</ul>
<h2 id="professional-trainings-and-certificates">Professional Trainings and Certificates</h2>
<ul>
<li>Microbial Genomics &amp; Metagenomics, Joint Genome Institute, 2018</li>
<li>Bioinformatics Genome Assembly, UC Davis, 2018</li>
<li>IBM Data Science Professional Certificate, 2018</li>
<li>Neural Networks and Deep Learning Certificate, 2017</li>
<li>Micro MBA, Rady School of Business, 2016</li>
<li>Innovation to Market, Rady School of Business, 2015</li>
</ul>
<h2 id="education">Education</h2>
<p><strong>Ph.D. in Marine Biology</strong><br>
University of California, San Diego<br>
2011 – 2018<br>
Research focus: Application of systems biology and bioinformatics on modeling marine microorganism pathway kinetics</p>
<p><strong>B.S. in Applied Chemistry</strong><br>
Ocean University of China, Qing Dao
2002 – 2006</p>
<h2 id="publications">Publications</h2>
<ul>
<li><strong>P1</strong>: Broddrick, J. T., Du N. (Co-1st author), et al. (2019). Cross-compartment metabolic coupling enables flexible photoprotective mechanisms in diatom Phaeodactylum tricornutum. New Phytologist.</li>
<li><strong>P2</strong>: Du N., et al. (2018). Simultaneous quantum yield measurements of carbon uptake and oxygen evolution in microalgal cultures. PLoS One.</li>
<li><strong>P3</strong>: Broddrick, J. T., et al. (2016). Unique attributes of cyanobacterial metabolism revealed by improved genome-scale metabolic modeling and essential gene analysis. PNAS, 113(51), E8344-E8353.</li>
<li><strong>P4</strong>: Woertz, I. C., et al. (2014). Life cycle GHG emissions from microalgal biodiesel–a CA-GREET model. Environmental Science &amp; Technology, 48(11), 6060-6068.</li>
</ul>

</body>
</html>
